### ISSUE TITLE
Make login action more easy to discover

### ISSUE THREAD

**Issue by mgallien** [nextcloud/desktop#3923]
When the desktop client is logged out from the server, the only way to log in again is via the menu related to the account accessible from the main dialog.
This is not necessarily easy to discover even though this action is critical to get the desktop client to work successfully again.
We may need to show it more prominently.
For example settings dialog has no way to access it even though we show the account has being disconnected.
For example the action may be shown directly without having to go through a menu when we are logged out.

**Comment by jancborchardt** [comment]
> We regularly get report of people confused by the exact way to log in again when the desktop client is disconnected.

Yeah, needing to find the "Log in" action is a bit undiscoverable.

How about: When you are logged out, and you select that account from the dropdown, on the top of the activity list in the tray menu we show either a row or taller empty content communicating:

> **Logged out**
> Log back in to resume sync
> (button saying "Log in")

**Comment by alfrenovsky** [comment]
I get a lot of complains "The cloud isn't working".

If the log out wasn't voluntary from the client the logged out status should be a warning status with some kind of alert.

**Comment by alfrenovsky** [comment]
There should be two different variables.
one with the user intention. (The user WANTS to be logged in or logged out) and other with the current login status. The system should take action if they differ. In this way involuntary logout will be resolved

**Comment by alfrenovsky** [comment]
C'mon !

**Comment by Rello** [comment]
do we still want to do this - or can we close it for the moment?

**Comment by jancborchardt** [comment]
@Rello yes – my recommendation from above still stands:

https://github.com/nextcloud/desktop/issues/3923#issuecomment-962995967
